ruby-challenges.rb

```diff
@@ -33,4 +59,13 @@
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
 
-# Pseudo code:
\ No newline at end of file
+# Pseudo code:
+
+def pal(word)
+    if word.downcase == word.downcase.reverse
+        "#{word} is a palindrome"
+    else "#{word} is not a palindrome"
+    end
+end
+
+p pal(palindrome_test_case1)
```

Comment:
Looks like you are missing the pseudocode for this problem and the last one.  Please update this PR with the pseudocode on these.